lib/audit_tables/base.rb

```diff
@@ -1,4 +1,6 @@
 # frozen_string_literal: true
+require 'active_record'
```

Comment:
The tests would pass if it required `lib/audit_tables` instead of explicitly requiring the file under test. However, that continued to fail when deployed to staging. What was occurring on staging doesn't actually make sense to me.